scripts/vpn-auto-stack.sh

```
@@ -0,0 +1,23 @@
+# Outputs: Exposes vpn_auto_reconnect_process_once and supporting helpers for daemons and installers.
+# Exit codes: None directly; sourced modules may exit/die during their operations.
+
+if [[ -n "${__VPN_AUTO_STACK_LOADED:-}" ]]; then
+  return 0
+fi
+__VPN_AUTO_STACK_LOADED=1
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# shellcheck source=scripts/vpn-auto-state.sh
+. "${SCRIPT_DIR}/vpn-auto-state.sh"
```

Shellcheck (0.11.0)

[info] Not following: scripts/vpn-auto-state.sh was not specified as input (see shellcheck -x).

(SC1091)